chainstory/chainstory/settings.py

```diff
@@ -127,24 +123,17 @@
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/4.0/howto/static-files/
 
-STATIC_URL = 'static/'
+STATIC_URL = '/static/'
+STATIC_ROOT = os.path.join(BASE_DIR, 'staticfiles')
```

Comment:
I guess whitenoise is just for serving html and what not?

Reply:
I actually do not recall, I'll look into it, but I think it is for being able to deal with static files in daphne specifically. Which I'm going to turn into svgs here eventually these pngs are the worst